### PR TITLE
Feat/getToken wrapper

### PR DESCRIPTION
**Pull request by aureldent**

### CODE REVIEW

File: lumapps/api/client.py [hunk 1]
```diff
@@ -190,6 +190,37 @@ def _prune(self, method_parts, content):
     def get_new_client_as(self, user):
         return ApiClient(self._auth_info, self.api_info, user=user)
 
+    def authenticate_on_behalf_of(self, user_email, customer_id):
+        """ Using a service account, authenticate as a user
+            (the service account has to be authorized to do so)
+
+            Can be used to reinstanciate a new ApiClient
```
Comment (ulodciv) [inline lumapps/api/client.py]: reinstanciate -> re-instantiate

**Comment by aureldent** [comment]
Because I wanted to have a function that does one and only one thing that is correctly authenticate as a new user.
We can maybe use that method to extend `get_new_client_as` if that seems right to you.
Also I feeled that authenticate_on_behalf_of was clearer.

**Comment by ulodciv** [comment]
I have nothing against creating a new method, but, doesn't `get_new_client_as` return also an `ApiClient` "on behalf of"?

**Comment by aureldent** [comment]
Never used it, but from what i understand not really that would use the account wide delegation (that use a little magic in the google library by calling the credentials `with_subject` method) so no quite what we want to do there :)

**Pull request merged**